### PR TITLE
EI new table and amendments

### PR DESCRIPTION
**Pull request by sarmaevanidfe**

### CODE REVIEW

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql [hunk 1]
```diff
@@ -93,19 +93,20 @@ VALUES
 INSERT INTO Mtd.SourceConfigForImport
 (SourceDatabaseName,SourceTableName,SourceSchemaName,ColumnNamesToInclude,ColumnNamesToExclude,ColumnNamesToMask,[ModelDataToPL])
 VALUES
- ('EmpInc','Accounts','dbo','[Id],[AccountLegalEntityId],[LegalEntityId],[HasSignedIncentivesTerms],[VrfVendorId],[VrfCaseId],[VrfCaseStatus],[VrfCaseStatusLastUpdatedDateTime]','[LegalEntityName]','',1)
+ ('EmpInc','Accounts','dbo','[Id],[AccountLegalEntityId],[LegalEntityId],[HashedLegalEntityId],[HasSignedIncentivesTerms],[SignedAgreementVersion],[VrfVendorId],[VrfCaseId],[VrfCaseStatus],[VrfCaseStatusLastUpdatedDateTime]','[LegalEntityName]','',1)
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:96]: Don't see HashedLegalEntityId added to any table?

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql [hunk 2]
```diff
@@ -93,19 +93,20 @@ VALUES
...
+,('EmpInc','ApprenticeshipIncentive','incentives','[Id],[AccountId],[ApprenticeshipId],[EmployerType],[StartDate],[IncentiveApplicationApprenticeshipId],[AccountLegalEntityId],[RefreshedLearnerForEarnings],[HasPossibleChangeOfCircumstances],[PausePayments],[SubmittedDate],[CourseName],[Status]','[FirstName],[LastName],[DateOfBirth],[ULN],[SubmittedByEmail]','[UKPRN]',0)
+,('EmpInc','ChangeOfCircumstance','incentives','[Id],[ApprenticeshipIncentiveId],[ChangeType],[PreviousValue],[NewValue],[ChangedDate]','','',0)
+,('EmpInc','ClawbackPayment','incentives','[Id],[ApprenticeshipIncentiveId],[PendingPaymentId],[AccountId],[AccountLegalEntityId],[Amount],[DateClawbackCreated],[DateClawbackSent],[CollectionPeriod],[CollectionPeriodYear],[SubNominalCode],[PaymentId]','','',0)
+,('EmpInc','CollectionCalendar','incentives','[Id],[PeriodNumber],[CalendarMonth],[CalendarYear],[EIScheduledOpenDateUTC],[CensusDate],[AcademicYear],[Active]','','',0)
+,('EmpInc','Learner','incentives','[Id],[ApprenticeshipIncentiveId],[ApprenticeshipId],[SubmissionFound],[SubmissionDate],[LearningFound],[HasDataLock],[StartDate],[InLearning],[CreatedDate],[UpdatedDate]','[ULN],[RawJSON]','[Ukprn]',0)
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:105]: The RawJson probably needs splitting and also would be good to check if we are okay to Import ULN and RawJson Security wise.

**Review by uhimabindu — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql (hunk 2 above)
Comment (sarmaevanidfe) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:105]: In the earlier implementation , we decided not to import the RawJson to PL and we are omitting ULN across all tables.

Comment (uhimabindu) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:105]: My Bad, was thinking it's in the ColumnNamesToMask list. It's all good then.

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql (hunk 1 above)
Comment (uhimabindu) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:96]: And It probably should be in Columns to Mask list.

**Review by uhimabindu — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportAccountsToPL.sql [hunk 3]
```diff
@@ -116,7 +116,8 @@ BEGIN TRY
 							,Acc_AccLegalEntity.SignedAgreementId
 							,Acc_AccLegalEntity.PendingAgreementVersion
 							,Acc_AccLegalEntity.PendingAgreementId
-							,Acc_AccLegalEntity.Deleted		
+							,Acc_AccLegalEntity.Deleted
+							,EI_Acc.HashedLegalEntityId
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportAccountsToPL.sql:120]: Does this field still need to be there?

**Review by uhimabindu — CHANGES_REQUESTED** [review]

**Review by uhimabindu — APPROVED** [review]

**Pull request merged**